src/app/editors/editors.component.ts

```diff
@@ -2,6 +2,8 @@ import {Component, Input, Output, EventEmitter} from '@angular/core';
 import {Action} from "../action";
 import {FileConfig} from "../file-config";
 import {StateService} from "../state.service";
+//momo
+import { AutorunControlInterface } from "../exercise/autoruncontrol.interface"
```

Comment:
@kirjs , removed.  Note the interface has a 'running' attribute that might go away if one of the TODO's--passing in an (@Input)run() method from `exercise,component ` sits well with you.  Thoughts welcome.